srcpkgs/po4a/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'po4a'
 pkgname=po4a
-version=0.54
+version=0.55
 revision=1
-build_style="perl-ModuleBuild"
 noarch=yes
+build_style="perl-ModuleBuild"
```

Comment:
no need to be quoted

Reply:
Fixed. Sorry, I tend to overlook relics like that. ;)